### DISCUSSION TITLE
Congratulations on your software "QLog".

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#1] [General]
Hello,

purely by chance I came across your project, installed the software on my Linux system and also already started to try out the software.

I must say that I am very positively surprised. Currently I use CQRLOG on Linux. With your software some things are solved differently and it takes some time to form an opinion.

But I am looking forward to watch the further development of QLog and wish you good luck.

73, Wolfgang

**Comment by foldynl** [comment]
Hello Wolfgang,

thank you very much for you feedback.

I also used CQRLOG for a long time, but unfortunately, its development froze and after several conversations with Petr (ok2cqr) I decided that it was time to move on to something new. However, there are not many logs for Linux. That's why I decided to do something new. Based on the good work of my predecessor, I continue to expand QLog.

Yes, you're right, there are many things solved differently in QLog. I am trying a modern design of the logging.  I would like to do things differently therefore some things can be useful, some can be harmful.

Feedback and notes are welcomed.

73, Ladislav (OK1MLG)

**Comment by dl2ki** [comment]
Hi Ladislav,

I feel the same way about QRLOG. That's why I occasionally look for alternatives. There are some alternatives, but their development often seems to stand still. For this reason, it has been difficult for me to switch to another log. Moreover, I am only QRV in CW and do not need many things like digital modes. 

I am 67 years old, and unfortunately I can hardly participate in such a project. I only programme scripts in Python for my own use. 

So I could only give you unsorted general feedback from my point of view, if that's OK with you, but I don't want to influence your project. I am always open to new things and can adapt well to new things. That's what always makes things exciting. Maybe the switch to your software will come in the not too distant future.  

Communication via Github is always a bit awkward for me, but I might be able to formulate some thoughts under "Ideas".

73, Wolfgang
DL2KI

**Comment by dl2ki** [comment]
Hi Ladislav,

thank you very much for the new version 7.0. 
The implementation of Ant/Rig/Red profiles looks very good.

73, Wolfgang
DL2KI

**Reply by foldynl** [reply]
Hi Wolfgang,

thank you very much. Internally there are many changes. I hope there aren't many bugs. I tested it quite intensely, but you know - it is a software.

Unfortunately, I implemented only RIT / XIT. Split mode is not implemented. I have a problem with frequency control. My 7300 can't send the current VFO and it's quite difficult to determine the RX / TX frequency correctly. Well, maybe later.

73, Ladislav
